Since Go 1.22 the build will fail with CGO_ENABLED=0 and -buildmode=pie
since we enable -buildmode=pie by default, we need to enable CGO.

See: https://gitlab.alpinelinux.org/alpine/aports/-/issues/15809

diff -upr fq-0.10.0.orig/Makefile fq-0.10.0/Makefile
--- fq-0.10.0.orig/Makefile	2024-03-02 13:28:08.077086376 +0100
+++ fq-0.10.0/Makefile	2024-03-02 13:28:14.091342026 +0100
@@ -7,7 +7,7 @@ all: test fq
 .PHONY: always
 
 fq: always
-	CGO_ENABLED=0 go build -o fq -ldflags "${GO_BUILD_LDFLAGS}" ${GO_BUILD_FLAGS} .
+	go build -o fq -ldflags "${GO_BUILD_LDFLAGS}" ${GO_BUILD_FLAGS} .
 
 test: always testgo testjq testcli
 test-race: always testgo-race testjq testcli
